master/getting-started/kubernetes/installation/hosted/calico.yaml

```diff
@@ -106,10 +106,15 @@ spec:
         k8s-app: calico-node
       annotations:
         scheduler.alpha.kubernetes.io/critical-pod: ''
-        scheduler.alpha.kubernetes.io/tolerations: |
-          [{"key": "dedicated", "value": "master", "effect": "NoSchedule" },
-           {"key":"CriticalAddonsOnly", "operator":"Exists"}]
     spec:
+       tolerations:
```

Comment:
For `calico/node`, can I suggest we use the same tolerations as are used in GKE?

https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/calico-policy-controller/calico-node-daemonset.yaml#L151-L158

This lets calico/node run on ALL nodes in the cluster, no matter the toleration. I think this is desirable in 99% of cases.

Reply:
Seems like a good idea, I'll add the missing toleration.

Reply:
I've updated to match the GKE tolerations, but I did leave this guy in https://github.com/projectcalico/calico/pull/1634/files#diff-4d27ea570c64ec5841f1db3b92a55c33R176

Reply:
sgtm

Note that the comment is not quite right any more - still references the master.

Reply:
Ah right, this is to enable us to run on any Node, will update the comment.

Reply:
Do we need all this tolerations?
The following should be enough according to the [doc](https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/):
> An empty key with operator Exists matches all keys, values and effects which means this will tolerate everything.
```
tolerations:
- operator: "Exists"
```